lisa/transformers/disable_cloud_components.py

```diff
@@ -43,6 +44,13 @@ def _internal_run(self) -> Dict[str, Any]:
         sed = node.tools[Sed]
         ls = node.tools[Ls]
 
+        if (
```

Comment:
It should be another transformer. So they can be organized easily.

Reply:
Should I try including it in the existing package_installer transformer? Right now it is designed for RPMs and other package files. I originally didn't include it in this tranformer because it would require quite a bit of changes for it to make sense. Also if I create another tranformer named package_installer or something similar it would be confusing.

Reply:
That one is to install local packages. Maybe you can create a new one called `install_repo_packages`